src/VirtoCommerce.CatalogModule.Web/Scripts/blades/dynamicAssociations/dynamicAssociation-viewer.tpl.html

```diff
@@ -0,0 +1,62 @@
+<div class="blade-static" ng-if="blade.isExpanded">
+    <div class="form-group">
+        <div class="form-input __search">
+            <input placeholder="{{'platform.placeholders.search-keyword' | translate}}" ng-model="filter.keyword" ng-keyup="$event.which === 13 && filter.criteriaChanged()" style="width: 190px">
+            <button class="btn __other" style="position: relative;right: 45px;">
+                <i class="btn-ico fa fa-remove" title="Clear" ng-click="filter.resetKeyword();filter.criteriaChanged()"></i>
+            </button>
+        </div>
+        <p class="form-count">{{'platform.list.count' | translate}}: <span class="count">{{pageSettings.totalItems | number:0}}</span></p>
+    </div>
+</div>
+<div class="blade-static __bottom" ng-include="'$(Platform)/Scripts/common/templates/ok-cancel.tpl.html'"></div>
+<div class="blade-content" ng-class="{'__large-wide': blade.isExpanded, '__normal': !blade.isExpanded}">
+    <div class="blade-inner">
+        <div class="inner-block" ng-init="setGridOptions({
+                enableSorting: false,
+                showTreeRowHeader: false,
+                data: 'items',
+                rowTemplate: 'export-item-list.row.html',
+                rowHeight: 60,
+                infiniteScrollRowsFromEnd: 10,
+                columnDefs: [
+                        {
+                            name: 'imgSrc', displayName: 'export.blades.export-generic-viewer.labels.picture',
+                            enableColumnResizing: false, width: 60,
+                            cellTemplate: 'export-item-list-icon.cell.html'
+                        },
+                        { name: 'name', displayName: 'export.blades.export-generic-viewer.labels.name', cellTemplate: 'export-item-list-name.cell.html' },
+                    ]})">
+            <div class="table-wrapper" ng-if="items.length > 0">
+                <div ui-grid="gridOptions" ui-grid-auto-resize ui-grid-save-state ui-grid-selection ui-grid-resize-columns ui-grid-move-columns ui-grid-pinning ui-grid-height ui-grid-infinite-scroll></div>
+            </div>
+            <div class="note" ng-if="!items.length">{{ 'platform.list.no-data' | translate }}</div>
+        </div>
+    </div>
+</div>
+
+<script type="text/ng-template" id="export-item-list.row.html">
+    <div ng-click='grid.appScope.selectItem($event, row.entity)' ng-repeat="(colRenderIndex, col) in colContainer.renderedColumns track by col.uid"
+         ui-grid-one-bind-id-grid="rowRenderIndex + '-' + col.uid + '-cell'" class="ui-grid-cell"
+         ng-class="{'ui-grid-row-header-cell': col.isRowHeader, '__selected': row.entity.id===grid.appScope.selectedNodeId }"
+         role="{{col.isRowHeader ? 'rowheader' : 'gridcell' }}" ui-grid-cell>
+    </div>
+</script>
+
+<script type="text/ng-template" id="export-item-list-icon.cell.html">
```

Comment:
Do we need these?
Also please create independent templates, not override export ones, e.g. export-item-list.row.html

Reply:
done